src/map/options.ts

```diff
@@ -339,6 +339,11 @@ export class MapOptions extends OptionsGroup {
             }
         });
 
+        // Stop progation when pressing a key (Jupyter widget)
```

Comment:
```suggestion
        // Stop propagation of keydown events. This is required for the Jupyter integration, 
        // otherwise jupyter tries to interpret key press in the modal as its own input
```